service-workers/service-worker/resources/import-scripts-diff-resource-map-worker.js

```diff
@@ -0,0 +1,12 @@
+importScripts('/resources/testharness.js');
+
+let version = null,
+    echo_output = null;
+importScripts('import-scripts-version.py');
+// Once imported, the stored script should be loaded for subsequent importScripts.
+const expected_version = version;
+
+version = null;
+importScripts('import-scripts-version.py', 'import-scripts-echo.py?msg=test');
```

Comment:
It kind of seems like an accident of history that this test is both testing 'import the same script URL multiple times' and 'importScripts() with multiple arguments' that #2613 is about. A simpler test for #2613 would just be a service worker that does a single importScripts(script1, script2); I assume the reason you're using "version" and "echo" is that they are the only scripts that write to global variables so you can do assert_equals() after the call.

I guess that's fine... just a bit hard to understand why this test is written the way it is. Another option could be to extend import-scripts-echo.py to accept a string for the name of the output variable.

Reply:
@mattto Thanks for your comment. Did your means changing the line 10 to e.g. `importScripts('import-scripts-echo.py?msg=test1', 'import-scripts-echo.py?msg=test2');`?

Reply:
I was thinking the following is more clearly testing what #2613 asks for:

```
echo1  = null;
echo2 = null;
importScripts('import-scripts-echo.py?output=echo1&msg=test1', 'import-scripts-echo.py?output=echo2&msg=test2');?
assert_equals(echo1, 'test1');
assert_equals(echo2, 'test2');
```

It'd have to extend import-scripts-echo.py to have an `output` param.

Reply:
Updated in newest commit